src/AfriCC/EPP/Client.php

```diff
@@ -32,6 +32,7 @@ class Client
     protected $serviceExtensions;
     protected $ssl;
     protected $local_cert;
+    protected $pass_phrase;
```

Comment:
Since ssl stream context option is called `passphrase` adding underscore here, is rather non-intuitive.